PhysicsTools/NanoAOD/python/autoNANO.py

```diff
@@ -28,6 +28,8 @@ def expandNanoMapping(seqList, mapping, key):
     # L1 flavours: add tables through customize, supposed to be combined with PHYS
     'L1' : {'customize': 'nanoL1TrigObjCustomize'},
     'L1FULL' : {'customize': 'nanoL1TrigObjCustomizeFull'},
+    # Muon POG flavours : add tables through customize, supposed to be combined with PHYS
+    'MUPOG' : {'customize' : 'PrepMuonCustomNanoAOD'},
```

Comment:
please use `PhysicsTools/NanoAOD/custom_muon_cff.PrepMuonCustomNanoAOD` instead in order to avoid the import at higher level

Reply:
Done